src/tuneflow_py/plugins/isan_fusion/instruments.py

```
@@ -0,0 +1,339 @@
+                ticks_per_quarter,  # Quarter note
+                ticks_per_quarter // 4  # Sixteenth note (for fast passages)
+            ])
+            
+            # Create rhythmic patterns
+            if random.random() < 0.3:  # 30% chance of rest
+                current_tick += note_duration
+                continue
+            
+            # Select note with emphasis on scale degrees 1, 3, 5
+            weights = [3, 1, 2, 1, 3, 1, 1][:len(scale_notes)]
+            midi_note = random.choices(scale_notes, weights=weights)[0]
```

Ruff (0.14.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)